etcdserver/api/v3lock/v3lockpb/v3lock.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+package v3lockpb;
+
+import "gogoproto/gogo.proto";
+import "etcd/etcdserver/etcdserverpb/rpc.proto";
+
+// for grpc-gateway
+import "google/api/annotations.proto";
+
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+
+service Lock {
+  rpc Lock(LockRequest) returns (LockResponse) {
+      option (google.api.http) = {
+        post: "/v3alpha/lock/lock"
+        body: "*"
+    };
+  }
+  rpc Unlock(UnlockRequest) returns (UnlockResponse) {
```

Comment:
Can we add a verify or check interface? So user can verify if itself is still the locker owner or the other component is still the owner of the lock?